sklearn/metrics/cluster/supervised.py

```diff
@@ -814,8 +816,82 @@ def normalized_mutual_info_score(labels_true, labels_pred, max_n_classes=5000):
     return nmi
 
 
+def fowlkes_mallows_score(labels_true, labels_pred, max_n_classes=5000):
+    r"""Mesure the similarity of two clusterings of a set of points.
+
+    The Fowlkes-Mallows index (FMI) is defined by the following formula::
+
+        FMI = TP / sqrt((TP + FP) * (TP + FN))
+
+    With :math:`TP` is the number of pair of points that belongs in the same
```

Comment:
I would explicitly write  'True Positive', 'False Negative', 'False Positive', since the notations are based on these names:
```Where :math:`TP` is the number of 'True Positive' (i.e. the number of pair of points that ...```